src/ext/java/org/opentripplanner/ext/transmodelapi/TransmodelAPI.java

```diff
@@ -153,6 +153,7 @@ public Response getGraphQL(
   @POST
   @Path("/graphql/batch")
   @Consumes(MediaType.APPLICATION_JSON)
+  @Deprecated
```

Comment:
I would even be fine if that was removed. Batch queries have dubious value in the era of HTTP2.

Reply:
I will remove it in a separate PR